src/commands/crashes/category.txt

```diff
@@ -0,0 +1 @@
+Commands related to Crashes
```

Comment:
We'll modify this text once we'll add more commands. But in the next 6 months it is unlikely we'll add more